examples/nivisa_dmm_measurement/measurement.py

```diff
@@ -101,29 +96,27 @@ def measure(
 
         resource_manager = create_visa_resource_manager(service_options.use_simulation)
         session_info = reservation.session_info[0]
-        session = stack.enter_context(
-            create_visa_session(resource_manager, session_info.resource_name)
-        )
-
-        # Work around https://github.com/pyvisa/pyvisa/issues/739 - Type annotation for Resource
-        # context manager implicitly upcasts derived class to base class
-        assert isinstance(session, pyvisa.resources.MessageBasedResource)
-
-        log_instrument_id(session)
-
-        # When this measurement is called from outside of TestStand (session_exists == False),
-        # reset the instrument to a known state. In TestStand, ProcessSetup resets the instrument.
-        if not session_info.session_exists:
-            reset_instrument(session)
-
-        function_enum = str_to_enum(FUNCTION_TO_ENUM, measurement_type)
-        resolution_value = str_to_enum(RESOLUTION_DIGITS_TO_VALUE, str(resolution_digits))
-        session.write("CONF:%s %.g,%.g" % (function_enum, range, resolution_value))
-        check_instrument_error(session)
-
-        response = session.query("READ?")
-        check_instrument_error(session)
-        measured_value = float(response)
+        with create_visa_session(resource_manager, session_info.resource_name) as session:
+            # Work around https://github.com/pyvisa/pyvisa/issues/739 - Type annotation for Resource
+            # context manager implicitly upcasts derived class to base class
+            assert isinstance(session, pyvisa.resources.MessageBasedResource)
+
+            log_instrument_id(session)
+
+            # When this measurement is called from outside of TestStand (session_exists == False),
+            # reset the instrument to a known state.
+            # In TestStand, ProcessSetup resets the instrument.
```

Comment:
Why did you split this line? It's not >100 columns.

Reply:
There are 102 columns and I encountered a lint error hence I moved it to the next line.

Reply:
Oops, I was looking at the wrapped version. 

Wrapping the last word is sufficient. The VS Code "Rewrap" extension is good for this.